issue_templates.md

```diff
@@ -0,0 +1,127 @@
+# Feature Request
+name: Feature Request
+
+about: Suggest an idea for this project
+
+title: "[FEATURE]"
+
+labels: 'feature request'
+
+**Is your feature request related to a problem? Please describe.**
+
+[A clear and concise description of what the problem is. Ex. I'm always frustrated when [...] ]
+
+**Describe the solution you'd like**
+
+[A clear and concise description of what you want to happen.]
+
+**Describe alternatives you've considered**
+
+[A clear and concise description of any alternative solutions or features you've considered.]
+
+**Additional context**
+
+[Add any other context or screenshots about the feature request here.]
+
+# Bug Report
+name: Bug Report
+
+about: Create a report to help us improve
+
+title: "[BUG]"
+
+labels: 'bug'
+
+
+**Describe the bug**
+
+[A clear and concise description of what the bug is.]
+
+**To Reproduce**
+
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+
+[A clear and concise description of what you expected to happen.]
+
+**Screenshots**
+
+[If applicable, add screenshots to help explain your problem.]
+
+**Environment (please complete the following information):**
+- OS: [e.g. Fedora, RHEL, CentOS, Ubuntu, Debian, Windows, MacOS, Linux]
+- Browser [e.g. chrome, safari]
+- Version [e.g. 22]
```

Comment:
Do we really need browser and version? What about which bash version they are using instead?